clang/test/SemaHLSL/BuiltIns/frac-errors.hlsl

```diff
@@ -0,0 +1,27 @@
+
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm -disable-llvm-passes -verify -verify-ignore-unexpected
+
```

Comment:
What will happen for 
```
struct A { float a; }; 
A fa;
__builtin_hlsl_elementwise_frac (fa)
```

Reply:
error: 1st argument must be a vector, integer or floating point type (was 'A')

Reply:
Just curious.
Don't need a test case I think.